### PR TITLE
remove dependency on neat-interpolation

### PR DESCRIPTION
**Pull request by larskuhtz**
Neat-interpolation doesn't consider the formatting of output as part of the stable API. Version 0.4 introduced white space change that broke genesis header hashes.

This PR drops the dependency on neat-interpolation and replaces its use by `unlined`, `intercalate` and `<>` from the `text` package that can be assumed to be stable.

### CODE REVIEW

File: test/Chainweb/Test/Pact/Checkpointer.hs [hunk 1]
```diff
@@ -663,62 +661,63 @@ toTerm' :: ToTerm a => a -> Term Name
...
+    , "      \"content\": content"
+    , "      })"
+    , "  )"
+    , "  (defun read-table ()"
+    , "    (select test-table (constantly true))"
+    , "  )"
+    , ")"
+    , ""
+    , "(create-table test-table)"
+    ]
```
Comment (emilypi) [inline test/Chainweb/Test/Pact/Checkpointer.hs:722]: We'll need to confirm with a full replay that this works. Pact is white-space aware when it generates its hashes. Any substantive change will mess things up.

File: tools/ea/Ea.hs [hunk 2]
```diff
@@ -170,38 +170,39 @@ genPayloadModule' v tag cwTxs =
                     execNewGenesisBlock noMiner (V.fromList cwTxs)
 
             let payloadYaml = TE.decodeUtf8 $ Yaml.encode payloadWO
-                modl = T.unlines $ startModule tag <> [payloadYaml] <> endModule
```
Comment (emilypi) [inline tools/ea/Ea.hs:173]: why this change?

File: test/Chainweb/Test/Pact/Checkpointer.hs (hunk 1 above)
Comment (larskuhtz) [reply test/Chainweb/Test/Pact/Checkpointer.hs:722]: This is test code, and the test still passes.

The only non-test change is to the genesis payloads. There are golden tests for the genesis hashes. Any change to the payloads should trigger those tests to fail.

File: tools/ea/Ea.hs (hunk 2 above)
Comment (larskuhtz) [reply tools/ea/Ea.hs:173]: This is the change that causes the generated payload modules to not depend on neat-interpolation. The code replaces the use of the QuasiQuote with `T.unlines`, `T.intercalate`, and `<>` for `Text`, which are expected to be stable.

File: test/Chainweb/Test/Pact/Checkpointer.hs (hunk 1 above)
Comment (emilypi) [reply test/Chainweb/Test/Pact/Checkpointer.hs:722]: Ah right, the goldens! They're finally good for something 😄

**Review by emilypi — APPROVED** [review]
